assets/src/application/ui/display/espressoIcons/collection/Copy.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import { EspressoIconProps, Icon } from '../types';
+import EspressoIcon from '../EspressoIcon';
+
+const Copy = React.forwardRef<SVGSVGElement, Partial<EspressoIconProps>>((props, ref) => (
+	<EspressoIcon {...props} name={Icon.COPY} ref={ref} />
+));
+
+Copy.displayName = 'Copy';
```

Comment:
yes, this will be helpful in dev tool, to avoid those anonymouse functions 
:+1: